src/components/views/dialogs/InteractiveAuthDialog.js

```diff
@@ -77,7 +77,7 @@ export default React.createClass({
                     <div role="alert">{ this.state.authError.message || this.state.authError.toString() }</div>
                     <br />
                     <AccessibleButton onClick={this._onDismissClick}
-                        className="mx_UserSettings_button"
+                        className="mx_GeneralButton"
```

Comment:
What is a `GeneralButton`? Could it just be `mx_Button`?

Reply:
A `GeneralButton` is a button that was using the user settings classes and I didn't want to break anything, so included the extra styles. I haven't tested, but I have a burning feeling that trying to make it a `Button` will be more effort than it's worth for this PR.

Reply:
Okay, fair enough. Maybe make a future issue to clean up / simplify button classes?